src/lib/Fullscreen.js

```diff
@@ -111,15 +111,22 @@ class Fullscreen extends EventEmitter {
      * Focuses the element
      *
      * @private
-     * @param {HTMLElement|Event} el - Fullscreen element or event
+     * @param {HTMLElement|Event} elementOrEvent - Fullscreen element or event
      * @return {void}
      */
-    focusFullscreenElement = (el) => {
+    focusFullscreenElement = (elementOrEvent) => {
         // Focus on the fullscreen element so keyboard
         // events are triggered without an extra click
         // If el has target property, then it is an Event
         // otherwise it is a HTMLElement
-        const element = el.target || el;
+        let element;
+        if (elementOrEvent.target) {
+            // This is an element
+            // On certain browsers (Firefox), the actual target is on the activeElement property of the event target
+            element = elementOrEvent.target.activeElement || elementOrEvent.target;
+        } else {
```

Comment:
You could simplify this a bit by avoiding the `else` clause here.

```
let element = elementOrEvent;

// Comment explaining why this may sometimes be an event instead of an element
if (element.target) {
    element = // etc.
}

element.focus();
```